project.clj

```diff
@@ -7,12 +7,21 @@
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0.html"}
 
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/java.jdbc "0.3.2"]
-                 [org.postgresql/postgresql "9.3-1100-jdbc41"]
-                 [com.jolbox/bonecp "0.8.0.RELEASE" :exclusions [[org.slf4j/slf4j-api]]]
+  :dependencies [[com.jolbox/bonecp "0.8.0.RELEASE" :exclusions [[org.slf4j/slf4j-api]]]
+                 [com.zaxxer/HikariCP-java6 "2.3.7" :exclusions [[org.slf4j/slf4j-api]]]
+                 [metrics-clojure "2.5.1" :exclusions [[org.slf4j/slf4j-api]]]
+                 [org.clojure/clojure "1.6.0"]
+                 [org.clojure/java.jdbc "0.3.6"]
+                 [org.postgresql/postgresql "9.4-1201-jdbc41"]
+                 [prismatic/schema "0.4.2"]
                  [puppetlabs/kitchensink ~ks-version]]
 
+  :profiles {:dev {:dependencies [[puppetlabs/trapperkeeper "1.1.1"]
+                                  [puppetlabs/trapperkeeper "1.1.1" :classifier "test"]]}}
```

Comment:
Yeah, the log silencing code is in trapperkeeper tests ... not sure if we have a lighter dependency we can pull in instead.